### PR TITLE
Added POST Formatted Transcript action

### PR DESCRIPTION
**Pull request by mjabali**

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    contentType: {
```
Comment (vellames-turing) [inline components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs:18]: Maybe we can move this prop to `app` in order to reuse it in future actions?

**Review by vellames-turing — CHANGES_REQUESTED** [review]
Looks good, just one minor suggestion

File: components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs (hunk 1 above)
Comment (mjabali) [reply components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs:17]: Cool... That's updated in e3e6191f. Thanks for the suggestion.

File: components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import symblAIApp from "../../symbl_ai.app.mjs";
-import constants from "../constants.mjs";
 
 export default {
   key: "symbl_ai-post-formatted-transcript",
   name: "Create Formatted Transcript",
   description: "Create formatted transcript from the Conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/transcript/).",
-  version: "0.0.1",
+  version: "0.0.8",
```
Comment (vellames-turing) [inline components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs:7]: I guess this is suppose to be `0.0.2`

**Review by vellames-turing — CHANGES_REQUESTED** [review]

File: components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs (hunk 2 above)
Comment (mjabali) [reply components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs]: It's actually 0.0.1. Thanks for pointing that out. Fixed in 81b88125.

**Review by vellames-turing — APPROVED** [review]

**Pull request merged**